src/simplewallet/simplewallet.cpp

```diff
@@ -4443,6 +4444,16 @@ bool simple_wallet::status(const std::vector<std::string> &args)
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::wallet_info(const std::vector<std::string> &args)
+{
+  success_msg_writer() << "Filename: " << m_wallet->get_wallet_file();
+  success_msg_writer() << "Address: " << m_wallet->get_account().get_public_address_str(m_wallet->testnet());
+  success_msg_writer() << "Watch only: " << (m_wallet->watch_only() ? "Yes" : "No");
+  success_msg_writer() << "Restricted RPC: " << (m_wallet->restricted() ? "Yes" : "No");
+  success_msg_writer() << "Testnet: " << (m_wallet->testnet() ? "Yes" : "No");
+  return true;
```

Comment:
You probably want the normal msg_writer here. Also please use tr("xxx") for strings so they can be translated. I wonder whether this is becoming a bit pointless now, but it's trivial change so...
Also, restricted RPC does not make sense in simplewallet.

Reply:
Thanks for the fast feedback. Almost as many issues as new code lines ... but I'm learning!

Updated.